Makefile

```diff
@@ -117,3 +137,11 @@ release: clean fmtcheck cross
 	@mkdir -p $(BUILD_DIR)/crc-$(CRC_VERSION)-linux-amd64
 	@cp LICENSE $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/crc-$(CRC_VERSION)-linux-amd64
 	tar cJSf $(RELEASE_DIR)/crc-$(CRC_VERSION)-linux-amd64.tar.xz -C $(BUILD_DIR) crc-$(CRC_VERSION)-linux-amd64
+
+.PHONY: embed_bundle
+embed_bundle: LDFLAGS += $(BUNDLE_EMBEDDED)
+embed_bundle: cross binappend
+	@$(call check_defined, BUNDLE_DIR, "Embedding bundle requires BUNDLE_DIR set to a directory with CRC bundles for all architectures")
```

Comment:
Error message `Embedding bundle requires BUNDLE_DIR set to a directory with CRC bundles for all architectures` is not clear `CRC bundles for all architectures` doesn't make sense it might be `CRC bundles for all hypervisors` ? 

`Embedding bundle requires BUNDLE_DIR set to a directory which contains CRC bundles for all hypervisors` might be better wdyt? cc @gbraad

Reply:
Right, architectures are mostly relating to `ia86 (x86)` or `ARM`, etc